### ISSUE TITLE
 RKE2 and K3S - KDM Update for K8s January patch release for Rancher 2.7

### ISSUE THREAD

**Issue by rancher-max** [rancher/rancher#40499]
# KDM Updates

| New Kubernetes Version  | Min Rancher Version | Max Rancher Version |
|                  :---:                  |              :---:               |               :---:               |
| v1.23.16+rke2r1                  | v2.6.4-alpha1             | v2.7.99 |
| v1.24.10+rke2r1                  | v2.6.7-alpha1             | v2.7.99 |
| v1.25.6+rke2r1                  | v2.7.2-alpha1             | v2.7.99 |
| v1.23.16+k3s1                   | v2.6.3-alpha1             | v2.7.99 |
| v1.24.10+k3s1                    | v2.6.7-alpha1             | v2.7.99 |
| v1.25.6+k3s1                  | v2.7.2-alpha1             | v2.7.99 |

# Chart Changelog

| Chart Name  | New Version | Affected Kubernetes Version(s) |
|        :---:        |       :---:        |                     :---:                     |
| rancher-vsphere-cpi | 1.4.101 | v1.23.16+rke2r1, v1.24.10+rke2r1 |
| harvester-cloud-provider | 0.1.1400 | v1.23.16+rke2r1, v1.24.10+rke2r1 |
| harvester-csi-driver | 0.1.1500 | v1.23.16+rke2r1, v1.24.10+rke2r1 |
| rke2-multus | v3.9.3-build2023010901 | v1.24.10+rke2r1 |

# Argument Changelog

| Argument Name  | Affected Kubernetes Version(s) | Value(s) |
|        :---:           |                     :---:                     | :---: |
| profile | v1.25.6+rke2r1 | cis-1.23 |

# Additional Notes
This is the first introduction of v1.25 into Rancher, so there is no "Chart Changelog" as they all are effectively new additions.

**Comment by rancher-max** [comment]
This was validated during release testing. The profile arg being updated in v1.25 was done later as part of an ongoing effort to make the switch from PSPs to PSAs between v1.24- to v1.25+. 

Closing this issue out due to the validations and because it will be superseded by February releases in a few weeks.

**Comment by snasovich** [comment]
This is waiting on https://github.com/rancher/kontainer-driver-metadata/pull/1089 before it can be tested for v2.7 on https://github.com/rancher/kontainer-driver-metadata/tree/dev-v2.7-for-2.6.11 branch.

**Comment by rancher-max** [comment]
This has been validated previously, and confirmed that v1.25 is now removed:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/55997940/221688715-2bf88576-73a2-4762-b73c-13907d62d4fe.png">

It shows experimental, which I believe is being handled by https://github.com/rancher/rancher/issues/40694
